packages/text-document-utils/.gitignore

```diff
@@ -0,0 +1,2 @@
+!test/test-data/module-loader/@scope/module/1.2.3/node_modules
```

Comment:
This also seems redundant, I don't see such files/folders in the package.

Reply:
yes, it was not needed at all,.